doomrainbow/src/main/java/com/detroitlabs/kyleofori/doomrainbow/RangeLabelAlignment.java

```diff
@@ -0,0 +1,5 @@
+package com.detroitlabs.kyleofori.doomrainbow;
+
+public enum RangeLabelAlignment {
+    INWARD, OUTWARD, CENTERED
```

Comment:
Me neither... unless we just allow the labels to be aligned left,right, and center individually.